platforms/common/cpp/core/JitProgram.cpp

```diff
@@ -5,12 +5,11 @@
 
 #include <iomanip>
 #include <sstream>
+#include <tuple>
```

Comment:
Will this compile for iOS?
It is std 98 there AFAIR.

Reply:
It currently doesn't need to be compilable for Android & iOS.

Reply:
JIT works only for x86, so this file won't be processed for iOS. If you meant MacOS, there is a bunch of places with e.g. unordered_map, and there is a "c++11" key in project file.